tests/test_hf_api.py

```diff
@@ -2981,8 +2981,10 @@ def test_list_repo_likers(self) -> None:
         likers = self.api.list_repo_likers(repo_id, token=TOKEN)
 
         # Check if the test user is in the list of likers
-        liker_usernames = [user.username for user in likers]
-        self.assertGreater(len(likers), 0)
+        likers_list = list(likers)
+        self.assertGreater(len(likers_list), 0)
+
+        liker_usernames = [user.username for user in likers_list]
```

Comment:
Regarding the test, we could use the production endpoint instead of testing on staging. For example, the [open_llm_leaderboard Space](https://huggingface.co/spaces/open-llm-leaderboard/open_llm_leaderboard) is the most liked repo on the Hub so results are for sure paginated.